Task2/Project/Shapes/Models/Vertex.cs

```diff
@@ -1,8 +1,77 @@
 namespace Shapes.Models
 {
+    /// <summary>
+    /// Represents one of points for creating the shape objects.
+    /// </summary>
     [System.Serializable]
     public class Vertex : ShapeBase
     {
-        public System.Windows.Point Location { get; set; }
+        // CONST
+        private const int RADIUS = 5;
+        // FIELDS
+        private System.Windows.Point location;
+        /// <summary>
+        /// Vertices counter.
+        /// </summary>
+        public static int CountVertices;

```

Comment:
where did you use it?

Reply:
We have different value in development brunch, you should check it out. probably you also need to merge development with your brunch

Reply:
done